catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -143,20 +144,20 @@ protected Object executeWithSubject() throws Exception {
         final File outputFile = initOutputFile(output);
         if (outputFile.exists()) {
             printErrorMessage(String.format("File [%s] already exists!", outputFile.getPath()));
-            return null;
```

Comment:
Absolutely. I'd rather use exceptions as a means of exceptional state than trying to parse log messages? Unless you have any other ideas.